WoD/UpperBlackrockSpire/Locales/zhCN.lua

```diff
@@ -1,6 +1,6 @@
-local L = BigWigs:NewBossLocale("Orebender Gor'ashan", "zhCN")
+﻿local L = BigWigs:NewBossLocale("Orebender Gor'ashan", "zhCN")
 if not L then return end
 if L then
-	--L.counduitLeft = "%d |4Conduit:Conduits; left"
+	L.counduitLeft = "%d |4导管:导管; 剩余"
```

Comment:
Since Chinese doesn't make a distinction between singular and plural, you can completely remove the special `|4` code.

It is perfectly fine to turn this line into the following: `"%d 导管 剩余"`

You can do this for any future translations.